src/providers/bedrock/complete.ts

```diff
@@ -754,3 +789,124 @@ export const BedrockCohereCompleteStreamChunkTransform: (
     ],
   })}\n\n`;
 };
+
+interface BedrocMistralStreamChunkOutput {
+  text: string;
+  stop_reason: string | null;
+}
+
+export interface BedrocMistralStreamChunk {
+  outputs: BedrocMistralStreamChunkOutput[];
+}
+
+export const BedrockMistralCompleteStreamChunkTransform: (
+  response: string,
+  fallbackId: string
+) => string | string[] = (responseChunk, fallbackId) => {
+  let chunk = responseChunk.trim();
+  chunk = chunk.trim();
+  const parsedChunk: BedrocMistralStreamChunk = JSON.parse(chunk);
+
+  if (parsedChunk.outputs[0].stop_reason) {
+    return [
+      `data: ${JSON.stringify({
+        id: fallbackId,
+        object: 'text_completion',
+        created: Math.floor(Date.now() / 1000),
+        model: '',
+        provider: BEDROCK,
+        choices: [
+          {
+            text: parsedChunk.outputs[0].text,
+            index: 0,
+            logprobs: null,
+            finish_reason: parsedChunk.outputs[0].stop_reason,
+          },
+        ],
+        usage: {
+          prompt_tokens: 0,
+          completion_tokens: 0,
+          total_tokens: 0,
+        },
+      })}\n\n`,
+      `data: [DONE]\n\n`,
+    ];
+  }
+
+  return `data: ${JSON.stringify({
+    id: fallbackId,
+    object: 'text_completion',
+    created: Math.floor(Date.now() / 1000),
+    model: '',
+    provider: BEDROCK,
+    choices: [
+      {
+        text: parsedChunk.outputs[0].text,
+        index: 0,
+        logprobs: null,
+        finish_reason: null,
+      },
+    ],
+  })}\n\n`;
+};
+
+interface BedrockMistralCompleteOutput {
+  text: string;
+  stop_reason: string;
+}
+
+export interface BedrockMistralCompleteResponse {
+  outputs: BedrockMistralCompleteOutput[];
+}
+
+export const BedrockMistralCompleteResponseTransform: (
+  response: BedrockMistralCompleteResponse | BedrockErrorResponse,
+  responseStatus: number
+) => CompletionResponse | ErrorResponse = (response, responseStatus) => {
+  if ('message' in response && responseStatus !== 200) {
```

Comment:
You can use `BedrockErrorResponseTransform` function to handle error for bedrock. 
https://github.com/Portkey-AI/gateway/blob/48785807bcc19dc13669354ac3bc895b9af58185/src/providers/bedrock/chatComplete.ts#L394-L405

Example usage: 
https://github.com/Portkey-AI/gateway/blob/48785807bcc19dc13669354ac3bc895b9af58185/src/providers/bedrock/complete.ts#L339-L344